### ISSUE TITLE
Create admin role to prevent users from accessing admin resources

### ISSUE THREAD

**Issue by lanesawyer** [lanesawyer/gshe#2]

**Comment by Stackranker** [comment]
Yes, for sure. We'll want to carefully control our data- I imagine we'll want to keep an accurate raw data source, then build most of our features and displays on top of filtered versions of/references to that raw data. 

Most users should only be contributing data such as new experiences, yes v. no answers, and A v. B answers, and their queries (such as comparisons to other users, their personal stackrank, etc.).
